src/app/test/helpers/faker.ts

```diff
@@ -0,0 +1,91 @@
+import { Id, ImageURL } from "@interfaces/apiInterfaces";
+import faker from "faker";
+
+export const modelData = {
+  boolean: () => faker.random.boolean(),
+  description: () => faker.lorem.sentence(),
+  id: (id?: Id) => (id ? id : faker.random.number(25)),
+  ids: () => randomArray(0, 5, () => faker.random.number(100)),
+  imageUrl: () => faker.image.imageUrl(),
+  imageUrls: () =>
+    new Array<ImageURL>(
+      {
+        size: "extralarge",
+        url: faker.image.imageUrl(300, 300),
+        width: 300,
+        height: 300,
+      },
+      {
+        size: "large",
+        url: faker.image.imageUrl(220, 220),
+        width: 220,
+        height: 220,
+      },
+      {
+        size: "medium",
+        url: faker.image.imageUrl(140, 140),
+        width: 140,
+        height: 140,
+      },
+      {
+        size: "small",
+        url: faker.image.imageUrl(60, 60),
+        width: 60,
+        height: 60,
+      },
+      {
+        size: "tiny",
+        url: faker.image.imageUrl(30, 30),
+        width: 30,
+        height: 30,
+      }
+    ),
+  latitude: () => parseFloat(faker.address.latitude()),
+  longitude: () => parseFloat(faker.address.longitude()),
+  notes: () => randomObject(0, 5),
+  offset: () =>
+    faker.random.arrayElement(["+", "-"]) +
+    faker.random.number(11) +
+    ":" +
+    faker.random.arrayElement(["00", "30"]),
+  param: () => faker.name.title(),
+  seconds: () => faker.random.number(86400 - 30) + 30,
+  timestamp: () => faker.date.past().toISOString(),
+  timezone: () => undefined, // TODO
```

Comment:
depending on where this is used it'll be something like `australia/sydney'

Reply:
My goal is to use timezones which are different to the one the machine is currently in. That way i can properly test conversions are done.